### DISCUSSION TITLE
Institute Details

### DISCUSSION THREAD

**Discussion by TextZip** [arc-bphc/arc-bphc.github.io#9] [Ideas]
@arc-bphc/core-committee  add BITS logo and Institute name at the top of the ARC logo with a hyperlink ..?? we need to mention our institute somewhere ig

**Comment by divyekalra1** [comment]
We could simply mention them in the **About Us** section of our landing page, with a link that redirects to the main bits website.

**Reply by TextZip** [reply]
yea that works as well, but BITS logo is :heart:, like we have our spons ka logo in our page but not our university

**Reply by divyekalra1** [reply]
There's a perfect place for the BITS logo in the footer though. We could add **"BITS Pilani, Hyderabad Campus"** and the logo to the right of the **"Automation and Robotics Club"** text with a vertical bar separating them.

**Reply by TextZip** [reply]
cool that and about us mey ek line would be perfect
